boards/firechip/base-workloads/br-base/overlay/etc/init.d/rcS

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+
+# Start all init scripts in /etc/init.d
```

Comment:
What is this needed for?

Reply:
This fixes the hang after "Starting Network" for some reason. Its the same as the default script, but with a print statement added.

Its not really a fix, but I have noticed this making that issue go away with this change,